bindings/wasm/src/account/account_builder.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::account::wasm_account::WasmAccount;
```

Comment:
Nit: Usually the imports are separated by blocks. One block with imports from the standard library, one from other libs, and one within the same crate. 

When I merge your branched into mine I changed this, so don't worry, you don't have do it.